build.sbt

```diff
@@ -194,6 +194,21 @@ lazy val storageS3DynamoDB = (project in file("storage-s3-dynamodb"))
     )
   )
 
+lazy val deltaIcebergCompat = (project in file("delta-iceberg-compat"))
+  .dependsOn(core % "compile->compile;test->test;provided->provided")
+  .settings (
+    name := "delta-iceberg-compat",
```

Comment:
@zsxwing do you think this is a good name? 
compat or compatibility? 

and should we have "spark" in the name?